community/modules/compute/gke-job-template/templates/gke-job-base.yaml.tftpl

```diff
@@ -61,5 +61,20 @@ spec:
             cpu: ${millicpu_request}
           %{~ endif ~}
         %{~ endif ~}
+        %{~ if length(volumes) != 0 ~}
```

Comment:
This is the nittiest of nits, but I prefer `> 0` for this comparison despite the impossibility of negative values.

Reply:
Done.